packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/WalletConnectProposalModal.tsx

```
@@ -0,0 +1,158 @@
+            </Paragraph>
+
+            <Card margin={{ top: spacings.xs }} paddingType="small">
+                {['active', 'inactive', 'unsupported'].map(status => {
+                    const filteredNetworks = pendingProposal.networks.filter(
+                        network => network.status == status,
+                    );
+                    if (!filteredNetworks?.length) return null;
+
+                    return (
+                        <>
+                            <Text variant="tertiary">{`Requested networks - ${status}: `}</Text>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)